ci/docker-compose.yml

```diff
@@ -45,11 +46,40 @@ services:
 
   clickhouse:
     image: yandex/clickhouse-server:1.1.54388
-
     ports:
       - 8123:8123
       - 9000:9000
 
+  kudu-master:
+    image: cpcloud86/kudu:latest
+    networks:
+      default:
+        aliases:
+          - kudu
+    cap_add:
+      - SYS_TIME
+    ports:
+      - 7051:7051
+      - 8051:8051
+    environment:
+      KUDU_MASTER: "true"
+    volumes:
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
+
+  kudu-tserver:
+    image: cpcloud86/kudu:latest
+    cap_add:
+      - SYS_TIME
+    ports:
+      - 7050:7050
+      - 8050:8050
+    environment:
+      KUDU_MASTER: "false"
+    volumes:
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
```

Comment:
I'm afraid this docker-compose will run dozens of services in the near future :D

Reply:
I was thinking we should break up the impala image into `hadoop`, `hive`, and `impala`. Is that completely insane?